frontend/src/metabase/visualizations/visualizations/CartesianChart/CartesianChart.tsx

```diff
@@ -62,230 +48,21 @@ export function CartesianChart(props: VisualizationProps) {
   const legendItems = useMemo(() => getLegendItems(chartModel), [chartModel]);
   const hasLegend = legendItems.length > 1;
 
-  const openQuestion = useCallback(() => {
```

Comment:
Extracted to a separate hook which hopefully will make it easy to decouple Combo from Waterfall from Scatter later